app/src/main/java/com/example/bigowlapp/activity/SignUpPageActivity.java

```diff
@@ -62,7 +62,9 @@ protected void initialize() {
                 this.userPhone.setError("please enter a phone number");
                 this.userPhone.requestFocus();
             } else {
-                signUpViewModel.createUser(email, pass, userPhone, firstName, lastName)
+                //String userPhone2 = userPhone.replaceFirst("(\\d{3})(\\d{3})(\\d+)", "($1) $2-$3");  //Format to be used?
+                String formattedPhone = userPhone.replaceAll("[^+0-9]", "");
```

Comment:
As the minimum requirement, the regex should account that a "+" sign should be in the beginning of the phone number, and that the phone number should end with a number.

If you feel adventurous and have the time, then consider using Google's phone number library https://github.com/google/libphonenumber. It can format number in a specific way. Moreover, it can actually compute whether the phone number is actually a real phone number with an actual area code, and country code.

Reply:
Thanks! I looked it up but there's a huge amount of files so I'm a bit confused on how to use.
I did a quick fix for now